.coveragerc

```diff
@@ -4,7 +4,10 @@ omit =
 
 [report]
 omit =
+  cnxdb/contrib/pytest.py
+  cnxdb/contrib/testing.py
```

Comment:
I'm using the [Django usage of the term 'contrib'](https://pythonhosted.org/django_simple_feedback/ref/contrib/index.html).